### ISSUE TITLE
isort incorrectly processed my code

### ISSUE THREAD

**Issue by zengxs** [timothycrosley/isort#751]
Original code:
~~~py
# -*- coding: utf-8 -*-

# Define your item pipelines here
#
# Don't forget to add your pipeline to the ITEM_PIPELINES setting
# See: https://doc.scrapy.org/en/latest/topics/item-pipeline.html
from datetime import datetime
from .items import WeiboMblogItem

class WeiboMblogPipeline(object):
    def process_item(self, item, spider):
        if isinstance(item, WeiboMblogItem):
            item = self._process_item(item, spider)
        return item

    def _process_item(self, item, spider):
        item['inserted_at'] = datetime.now()
        return item
~~~

Sorted code:
~~~py
# -*- coding: utf-8 -*-


class WeiboMblogPipeline(object):
# Don't forget to add your pipeline to the ITEM_PIPELINES setting
# See: https://doc.scrapy.org/en/latest/topics/item-pipeline.html
from datetime import datetime

# Define your item pipelines here
#
from .items import WeiboMblogItem

    def process_item(self, item, spider):
        if isinstance(item, WeiboMblogItem):
            item = self._process_item(item, spider)
        return item

    def _process_item(self, item, spider):
        item['inserted_at'] = datetime.now()
        return item
~~~

isort version: 4.3.4
OS: macOS 10.13
Python: 3.6.6

**Comment by Siecje** [comment]
I was able to reproduce this with `VERSION 4.3.21` but it is fixed on `develop`.

**Comment by timothycrosley** [comment]
This is fixed in develop and will make it out in the 5.0.0 release.

Thanks!

~Timothy
